vector/src/main/java/im/vector/activity/VectorBaseSearchActivity.java

```diff
@@ -66,11 +67,19 @@
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        setContentView(R.layout.activity_vector_unified_search);
```

Comment:
This layout has nothing to do here. This activity should probably be declared abstract since it cannot be used directly. Calling setContentView() knowing that it'll be overriden by the activity extending this class doesn't feel right and is expensive.